examples/thisConsensusTime/index.rsh

```diff
@@ -0,0 +1,23 @@
+'reach 0.1';
+
+export const main = Reach.App(() => {
+  const A = Participant('Alice', {});
+  init();
+  A.publish();
+  commit();
+
+  A.publish();
+
+  var [ lastTime, lastSecs, i ] = [ thisConsensusTime(), thisConsensusSecs(), 0 ];
+  invariant(lastTime > lastConsensusTime() &&
```

Comment:
The way you did it, is it possible to prove something about the connection between `this` and `last`?

Reply:
I think that's all I mean... that you could put an invariant `thisTime > lastTime`